mlflow/server/js/src/components/ExperimentView.js

```diff
@@ -323,7 +323,7 @@ export class ExperimentView extends Component {
             <div className="ExperimentView-search-inputs">
               <div className="ExperimentView-search">
                 <div className="ExperimentView-search-input">
-                  <label className="filter-label">Search Expression:</label>
+                  <label className="filter-label">Expression:</label>
```

Comment:
I think you should just call this Search. It's not very common to write Expression for a search field. People can figure out the syntax by pressing the question mark button.